.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file.

Reply:
Generally this should be committed for shared settings. I can remove it if you feel strongly about it, but it's pretty standard.